src/Stratis.SmartContracts.CLR/CallDataSerializer.cs

```diff
@@ -163,6 +166,18 @@ protected void AddMethodParams(List<byte[]> rlpBytes, object[] methodParameters)
             }
         }
 
+        protected void AddSignatures(List<byte[]> rlpBytes, string[] signatures)
```

Comment:
It might be good to put a summary here to explain/detail the format of he signatures?

Reply:
Done.